plugin/input/journalctl/journalctl.go

```diff
@@ -0,0 +1,84 @@
+package journalctl
+
+import (
+	"github.com/ozonru/file.d/fd"
+	"github.com/ozonru/file.d/pipeline"
+)
+
+/*{ introduction
+Input plugin, that reads journalctl logs
+}*/
+type Plugin struct {
+	params *pipeline.InputPluginParams
+	config *Config
+	reader *journalReader
+	offset *offsetInfo
+}
+
+type Config struct {
+	//! config-params
+	//^ config-params
+
+	//> @3@4@5@6
+	//>
+	//> The filename to store offsets of processed messages.
+	OffsetsFile string `json:"offsets_file" required:"true"` //*
+
+	//> @3@4@5@6
+	//>
+	//> Additional args for journalctl
+	JournalArgs []string `json:"journal_args" default:""` //*
```

Comment:
what are the available options here?

Reply:
Should we add `-o json ` by default? Will the plugin work without these parameters?